nexus/peer-mysql/Cargo.toml

```diff
@@ -10,7 +10,7 @@ anyhow = "1.0"
 async-trait = "0.1"
 chrono.workspace = true
 futures = { version = "0.3.28", features = ["executor"] }
-mysql_async = { version = "=0.34.1", default-features = false, features = ["minimal-rust", "rust_decimal", "chrono", "rustls-tls"] }
+mysql_async = { version = "=0.34.2", default-features = false, features = ["minimal-rust", "rust_decimal", "chrono", "rustls-tls"] }
```

Comment:
@iamKunalGupta I thought renovate would stop trying to update to 0.34.2 after you closed last week's PR?

Reply:
Have added to ignore for now https://github.com/PeerDB-io/peerdb/commit/2dff106c02f2fafcb84f07c8cb21304ad601b9ae, can enable separately